lib/honeycomb/client.rb

```diff
@@ -76,6 +82,10 @@ def add_field_to_trace(key, value)
       context.current_span.trace.add_field("app.#{key}", value)
     end
 
+    def debug?
+      client.debug ? true : false
+    end
+
```

Comment:
same question here as above, what is the use case that you were seeing for exposing this?

Reply:
Removed.